podman/api_connection.py

```diff
@@ -13,20 +13,26 @@
 
 
 class ApiConnection(HTTPConnection, AbstractContextManager):
-    """ ApiConnection provides a specialized HTTPConnection
-        to a Podman service."""
-
-    def __init__(self, url, base="/v1.24/libpod", *args,
-                 **kwargs):  # pylint: disable-msg=W1113
+    """
+    ApiConnection provides a specialized HTTPConnection
+    to a Podman service.
+    """
+
+    def __init__(
+        self, url, base="/v1.24/libpod", *args, **kwargs
+    ):  # pylint: disable-msg=W1113
         if url is None or not url:
             raise ValueError("url is required for service connection.")
 
         super().__init__("localhost", *args, **kwargs)
         supported_schemes = ("unix", "ssh")
         uri = urllib.parse.urlparse(url)
-        if uri.scheme not in ("unix", "ssh"):
-            raise ValueError("The scheme '{}' is not supported, only {}".format(
-                uri.scheme, supported_schemes))
+        if uri.scheme not in supported_schemes:
+            raise ValueError(
+                "The scheme '{}' is not supported, only {}".format(
+                    uri.scheme, supported_schemes
+                )
+            )
```

Comment:
I personally prefer something like the previous format but I'm not against this way for readability.